app/assets/stylesheets/components/editor-toolbar.scss

```diff
@@ -1,11 +1,5 @@
 .editor-toolbar {
   display: flex;
-  background: var(--base-0);
```

Comment:
This wasn't really needed.

Reply:
yeah I think this is a hangover from the initial storybook implementation which didn't have a background otherwise, but I think that's since been added in the story itself 🧹